cura/CuraApplication.py

```diff
@@ -179,6 +179,7 @@ def __init__(self, *args, **kwargs):
         self._use_single_instance = False
 
         self._single_instance = None
+        self._project_mode = None
```

Comment:
```suggestion
        self._project_mode: Optional[str] = None
```

Nitpicking, but it's better to type this. Perhaps we should also rename this to "open_project_mode" as that's a bit more verbose. The current name implied to me that it's a boolean (eg "we are in project mode").

Reply:
fixed 30e4c61620fc31703cf0b4227f419beea882db30